tasks/styles.js

```diff
@@ -45,6 +46,9 @@ module.exports = (gulp, $, pkg) => {
         }),
         outputStyle: options.outputStyle
       }).on('error', reportError))
+      .pipe(function() {
```

Comment:
1. Can we skip the indirection through an anonymous function? Or why is it necessary?

1. Icons and fonts should be hashed as well. Can you add those?

```suggestion
      .pipe(imagehash('./dist/', ['eot', 'gif', 'ico', 'jpg', 'jpeg', 'otf', 'png', 'ttf', 'webp', 'woff', 'woff2']))
```